bugwarrior/db.py

```diff
@@ -197,6 +196,19 @@ def find_taskwarrior_uuid(tw, keys, issue):
             )
         )
 
+    # old bugwarrior did not consider completed task so there could be multiple ones matching
+    for service, key_list in six.iteritems(keys):
+        if any([key in issue for key in key_list]):
+            results = tw.filter_tasks({
+                'and': [[("%s.is" % key, issue[key]) for key in key_list], ('status', 'completed')]
```

Comment:
Looks like these lists need to be concatenated, so something like:

```py
               'and': [("%s.is" % key, issue[key]) for key in key_list] + ['status', 'completed'],
```

Reply:
looks better, thx. pushed.